lib/blueflood_http_client.js

```diff
@@ -157,6 +165,7 @@ function buildPayloads(tenantId, timestamp, gauges, counters, timers, sets, flus
           sets: []
         }
         curJsonSize = basePayloadLength;
+        objLength = JSON.stringify(metric).length;
```

Comment:
This line appears to overwrite the `objLength++` computation. Was that on purpose?